src/Xamarin.Android.Tools.AndroidSdk/AndroidVersions.cs

```diff
@@ -173,6 +173,10 @@ static bool MatchesId (AndroidVersion version, string id)
 			new AndroidVersion (28, "9.0",   "Pie") {
 				AlternateIds = new[]{ "P" },
 			},
+			new AndroidVersion (29, "10.0",  "Android X") {
+				AlternateIds = new[]{ "Q" },
+			},
+			new AndroidVersion (30, "11.0",   "R"),
```

Comment:
This should probably just omit the `codeName` parameter, and instead include `R` in the `AlternateIds`:

```csharp
new AndroidVersion (30, "11.0") {
    AlternateIds = new[]{ "R" },
},
```